lib/deck.rb

```diff
@@ -0,0 +1,16 @@
+class Deck
+  attr_accessor :cards
+  attr_reader :cards_in_category
+  def initialize(cards)
+    @cards = cards
+  end
+
+  def count
+    cards.count
```

Comment:
Be careful about referencing instance variables without using the `@` sign - it's totally ok and Ruby lets us do it _if_ you're also using an `attr_reader` or `accessor`. It's a good idea to just match them up every time with `@` to keep track of them, especially in bigger more complex programs.